dotCMS/src/main/java/com/dotmarketing/portlets/fileassets/business/FileAssetAPIImpl.java

```diff
@@ -591,15 +609,15 @@ else if(type.equals("application/x-bzip2")) {
         while (bytesRead < metadataLimitInBytes && (copied = input.read(buf,0,buf.length)) > -1 ) {
             baos.write(buf,0,copied);
             bytesRead = bytesRead + copied;
-            
+
         }
 
         InputStream limitedInput = new ByteArrayInputStream(baos.toByteArray());
 
         //let's close the original input since it's no longer necessary to keep it open
         if (input != null) {
             try {
-                input.close();
+                input.close(); // todo: the file resource close handling for io should be on try catch
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)